lib/shortcode.php

```diff
@@ -18,12 +18,24 @@ function anvato_shortcode_get_parameters( $attr ) {
 	$analytics = Anvato_Settings()->get_options( Anvato_Settings::ANALYTICS_SETTINGS_KEY );
 	$monetization = Anvato_Settings()->get_options( Anvato_Settings::MONETIZATION_SETTINGS_KEY );
 
+	if ( ! isset( $player['width'] ) || ! isset( $player['width_type'] ) ) {
+		// If either is unset, this can't be a valid value so substitute one.
+		$width = '100%';
+	} else {
+		$width = $player['width'] . $player['width_type'];
+	}
+	if ( ! isset( $player['height'] ) || ! isset( $player['height_type'] ) ) {
```

Comment:
@alwaysblank what is height type? is this like px or % ?
so it could be 100px or 100% ?

Reply:
@sdeckernexstar Correct, it's `%` or `px`. I checked a couple sites on `staging` and they had `100%` for both values, which is why I chose it as the default here. 

I _strongly suspect_ that the error only shows up on sites where the Anvato settings page has never been saved and thus no value exists in `options` for this stuff. 

I chose to set this default value if either "thing" (length or type) was unset, because if only one is set we don't have enough data to guess what the other one is without making a lot of assumptions (and introducing a lot more logic here). We could probably have a simple check like "if height is a value > 100 and height_type is unset, then set height_type to 'px', otherwise set it to '%'."